docs/source/write_first_app.rst

```diff
@@ -101,889 +97,485 @@ bringing up a new one.
 
-  // Create a new gateway instance for interacting with the fabric network.
-  // In a real application this would be done as the backend server session is setup for
-  // a user that has been verified.
-  const gateway = new Gateway();
+The sample application uses the Gateway connection to get a reference to the Network and then the default Contract
+within a chaincode deployed on that network.
 
-  try {
-    // setup the gateway instance
-    // The user will now be able to create connections to the fabric network and be able to
-    // submit transactions and query. All transactions submitted by this gateway will be
-    // signed by this user using the credentials stored in the wallet.
-    await gateway.connect(ccp, {
-      wallet,
-      identity: userId,
-      discovery: {enabled: true, asLocalhost: true} // using asLocalhost as this gateway is using a fabric network deployed locally
-    });
+.. code-block:: TypeScript
 
-    // Build a network instance based on the channel where the smart contract is deployed
-    const network = await gateway.getNetwork(channelName);
+  const channelName = 'mychannel';
+  const chaincodeName = 'basic';
 
+  const network = gateway.getNetwork(channelName);
+  const contract = network.getContract(chaincodeName);
 
-    // Get the contract from the network.
-    const contract = network.getContract(chaincodeName);
+When a chaincode package includes multiple smart contracts, on the `getContract() <https://hyperledger.github.io/fabric-gateway/main/api/node/interfaces/Network.html#getContract>`__
+call you can specify both the name of the chaincode and a specific smart contract to target. For example:
 
-When a chaincode package includes multiple smart contracts, on the `getContract() API <https://hyperledger.github.io/fabric-sdk-node/release-2.2/module-fabric-network.Network.html#getContract>`__ you can specify both the name of the chaincode package and a specific smart contract to target. For example:
+.. code-block:: TypeScript
 
-.. code:: bash
+  const contract = network.getContract('chaincodeName', 'smartContractName');
 
-  const contract = await network.getContract('chaincodeName', 'smartContractName');
+Fourth, the application initializes the ledger with some sample assets
+----------------------------------------------------------------------
 
-Fourth, the application initializes the ledger with some sample data
---------------------------------------------------------------------
+Now that we are at the point where we are actually invoking smart contract transactions, let’s
+go through them in sequence.
 
-Now that we are at the point where we are actually having the sample application submit transactions, let’s
-go through them in sequence. The application code snippets and invoked chaincode snippets
-are provided for each called function, as well as the terminal output.
+Immediately after initial deployment of the chaincode, the ledger is empty. The application uses ``submitTransaction()``
```

Comment:
Okay so this example "initiates" a brand new channel it seems. First transaction sent to the ledger right. Lets get some language at the top start that explains this full scenario and the e2e. If it's not there yet.

Reply:
No, absolutely not. Your question makes it sound like you have a fundamental mis-understanding about what a channel is.